multinet/multinet/__init__.py

```diff
@@ -15,14 +15,32 @@
 from .schema import schema
 from . import db
 
+
+def graphql_query(query):
```

Comment:
I like moving this into its own function.

Reply:
It would probably be to our benefit to use a mutation query through this endpoint to create/update tables/graphs based on the bulk loading endpoints. That way we can avoid having db-specific code in the handlers for those endpoints. If we do that, we should probably also consider graphql endpoints for adding nodes and lists of nodes... That is all probably outside the scope of this task, though.

In short, keep this even if we only use it in one place for now.

Reply:
See issue #25